moped-editor/src/views/projects/projectsListView/helpers.js

```diff
@@ -128,8 +128,11 @@ export const useTableComponents = ({
 /**
  * The Material Table column settings
  */
-export const useColumns = ({ hiddenColumns, linkStateFilters, classes }) =>
-  useMemo(
+export const useColumns = ({ hiddenColumns, classes }) => {
+  const location = useLocation();
+  const queryString = location.search;
+
+  const columns = useMemo(
```

Comment:
This is the source of the `queryString` the comes through the React Router Links that carry state to `ProjectView` so we can link back to the previous filters.